_posts/2024-05-23-pyos-pyconus-2024-recap.md

```diff
@@ -0,0 +1,258 @@
+
+### So different from last year
+
+My experience last year was a bit different. I walked into my first PyCon
+US - having no idea how I'd fit in at the meeting. I didn’t know
+many people. I had never led a [mentored sprint](https://www.mentored-sprints.dev/).
+And I wasn't sure how my packaging talk would be received; you see, packaging is a notoriously thorny
+topic.
+
+[Last year turned out pretty great - read more here.](pycon-2023-packaging-presentation-sprints-leah-wasser.html)
+
+However, rather than feeling like an outsider, I met a lot of new people
+who became new Python friends and colleagues. I learned about a lot of cool
+projects and organizations and our sprints were awesome with many new contributions. Yaas! <i class="fa-solid fa-rocket"></i>
+
+My takeaway: if you are considering going to PyCon but are worried about not knowing
+people YOU SHOULD STILL GO! You can feel the true spirit of open source (and open science) at PyCon US. No judgement. All "levels" of Pythonistas welcome.
+
+
+## My first main track PyCon US talk Friends don't let friends package alone
+
+This year, I gave my first main track talk at PyCon, titled “Friends Don’t Let
+Friends Package Alone.” Getting a main track slot means presenting on a big
+stage to a huge room of people! It was real - headset mic, incredible
+tech support, and even an “escort” from the speaker room to the main stage. Wow!
+
+Speaking of friends, it was my friends who got me through this talk. I saw them sitting in
+the front rows, smiling and supporting me. They both empowered me and gave me calm.
+
+As someone who often battles imposter syndrome, giving a talk on the "big" stage of a tech meeting
+was unforgettable.
+
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/may/leah-talking-pyconus-2024.jpg" alt="Image of a woman on a big stage wearing a red tank top talking. On the side is a slide that says pyOpenSci - Save the Date. Open science fall festival september 28-29 2024" style="width: 70%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> Me talking on the stage! I was so nervous before but once I got up there, everything calmed in my mind.  <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+### A talk about making Python packaging more beginner friendly
+My talk focused on how pyOpenSci helps beginners by breaking down
+complex packaging concepts into simple, digestible pieces. I leaned into decades of
+experience and study of teaching data science to various audiences, from big-ten university students
+to students at under-resourced and under-served <a href="https://www.leahwasser.com/lessons-learned-working-with-tribal-hispanic-serving-schools-earth-data-science-corps" target="_blank">tribal colleges</a>. The essence of my talk was about the importance of
+community and collaboration in the thorny Python packaging ecosystem. And also why documentation should focus on fundamental concepts that users need to understand rather than tools.
```

Comment:
```suggestion
community and collaboration in the thorny Python packaging ecosystem, and also why documentation should focus on fundamental concepts that users need to understand, rather than tools.
```